jenkins/ci.suse.de/templates/cloud-ardana-job-std-3cp-devel-staging-updates-template.yaml

```diff
@@ -12,14 +17,14 @@
 
     builders:
       - trigger-builds:
-        - project: openstack-ardana
+        - project: openstack-ardana-pipeline
```

Comment:
do we plan to keep openstack-ardana for something? otherwise I'd prefer to replace it just with the new implementation rather than renaming it..

Reply:
... and done